src/main/java/gregtech/api/unification/material/materials/OrganicChemistryMaterials.java

```diff
@@ -470,5 +470,18 @@ public static void register() {
                 .flags(GENERATE_GEAR, GENERATE_RING, FLAMMABLE, NO_SMASHING, DISABLE_DECOMPOSITION, GENERATE_FOIL, GENERATE_BOLT_SCREW)
                 .components(Carbon, 5, Hydrogen, 8)
                 .build();
+
+        Butraldehyde = new Material.Builder(1069, "butraldehyde")
+                .fluid()
+                .colorAverage()
+                .components(Propene, 1, Hydrogen, 2, CarbonMonoxide, 1)
+                .build();
```

Comment:
The formula for this should be manually set to `C4H8O`.
Also, shouldn't this be `Butyraldehyde`?

Reply:
Lets just do the components as `Carbon, 4, Hydrogen, 8, Oxygen, 1` and disable decomposition to be consistent. Also yes, it should be `Butyraldehyde`